public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ResourcePicker/NestedResourceTable.test.tsx

```diff
@@ -1,108 +0,0 @@
-import { act, render, screen } from '@testing-library/react';
```

Comment:
what happened to these tests?

Reply:
I removed them because I felt like we're adequately testing them at the Resource Picker level